Solutions/NXLogDnsLogs/Parsers/NXLog_parsed_DNS_Server_ASim_view.txt

```diff
@@ -0,0 +1,236 @@
+    , 264, 'Dynamic update'
+    , 265, 'Zone XFR'
+    , 266, 'Zone XFR'
+    , 267, 'Zone XFR'
+    , 268, 'Zone XFR'
+    , 269, 'Zone XFR'
+    , 270, 'Zone XFR'
+    , 271, 'Zone XFR'
+    , 272, 'Zone XFR'
+    , 273, 'Zone XFR'
+    , 274, 'Zone XFR'
+    , 275, 'Zone XFR'
+    , 276, 'Zone XFR'
+    , 277, 'Dynamic update'
+    , 278, 'Dynamic update'
+    , 279, 'Query'
+    , 280, 'Query'
+  ];
+  let EventSubTypeTable=datatable(EventOriginalType:real,EventSubType:string)[
+    256, 'request'
+  , 257, 'response'
+  , 258, 'response'
+  , 259, 'response'
+  , 260, 'request'
+  , 261, 'response'
+  , 262, 'response'
+  , 263, 'request'
+  , 264, 'response'
+  , 265, 'request'
+  , 266, 'request'
+  , 267, 'response'
+  , 268, 'response'
+  , 269, 'request'
+  , 270, 'request'
+  , 271, 'response'
+  , 272, 'response'
+  , 273, 'request'
+  , 274, 'request'
+  , 275, 'response'
+  , 276, 'response'
+  , 277, 'request'
+  , 278, 'response'
+  , 279, 'NA'
+  , 280, 'NA'
+  ];
+  let EventResultTable=datatable(EventOriginalType:real,EventResult:string)[
+      256, 'NA'
+    , 257, 'Success'
+    , 258, 'Failure'
+    , 259, 'Failure'
+    , 260, 'NA'
+    , 261, 'NA'
+    , 262, 'Failure'
+    , 263, 'NA'
+    , 264, 'Based on RCODE'
+    , 265, 'NA'
+    , 266, 'NA'
+    , 267, 'Based on RCODE'
+    , 268, 'Based on RCODE'
+    , 269, 'NA'
+    , 270, 'NA'
+    , 271, 'Based on RCODE'
+    , 272, 'Based on RCODE'
+    , 273, 'NA'
+    , 274, 'NA'
+    , 275, 'Success'
+    , 276, 'Success'
+    , 277, 'NA'
+    , 278, 'Based on RCODE'
+    , 279, 'NA'
+    , 280, 'NA'
+  ];
+  let RCodeTable=datatable(DnsResponseCode:int,ResponseCodeName:string)[
+      0,'NOERROR'
+    , 1,'FORMERR'
+    , 2,'SERVFAIL'
+    , 3,'NXDOMAIN'
+    , 4,'NOTIMP'
+    , 5,'REFUSED'
+    , 6,'YXDOMAIN'
+    , 7,'YXRRSET'
+    , 8,'NXRRSET'
+    , 9,'NOTAUTH'
+    , 10,'NOTZONE'
+    , 11,'DSOTYPENI'
+    , 16,'BADVERS'
+    , 16,'BADSIG'
+    , 17,'BADKEY'
+    , 18,'BADTIME'
+    , 19,'BADMODE'
+    , 20,'BADNAME'
+    , 21,'BADALG'
+    , 22,'BADTRUNC'
+    , 23,'BADCOOKIE'
+  ];
+  let QTypeTable=datatable(DnsQueryType:int,QTypeName:string)[
+      0, 'Reserved'
+    , 1, 'A'
+    , 2, 'NS'
+    , 3, 'MD'
+    , 4, 'MF'
+    , 5, 'CNAME'
+    , 6, 'SOA'
+    , 7, 'MB'
+    , 8 ,'MG'
+    , 9 ,'MR'
+    , 10,'NULL'
+    , 11,'WKS'
+    , 12,'PTR'
+    , 13,'HINFO'
+    , 14,'MINFO'
+    , 15,'MX'
+    , 16,'TXT'
+    , 17,'RP'
+    , 18,'AFSDB'
+    , 19,'X25'
+    , 20,'ISDN'
+    , 21,'RT'
+    , 22,'NSAP'
+    , 23,'NSAP-PTR'
+    , 24,'SIG'
+    , 25,'KEY'
+    , 26,'PX'
+    , 27,'GPOS'
+    , 28,'AAAA'
+    , 29,'LOC'
+    , 30,'NXT'
+    , 31,'EID'
+    , 32,'NIMLOC'
+    , 33,'SRV'
+  ];
+  DNS_Server_CL
+  | where EventID_d < 281
+  | project-rename
+      DnsFlags=Flags_s,
+      DnsQuery=QNAME_s,
+      DnsQueryType=QTYPE_s,
+      DnsResponseCode=RCODE_s,
+      DnsResponseName=PacketData_s,
+      Dvc=Hostname_s,
+      DvcIpAddr=Host_IP_s,
```

Comment:
Please make the column name consistent. In Sample data it is HostIP_s and here it is Host_IP_s.

Reply:
Good catch!  Corrected in commit [3cd9e96](https://github.com/Azure/Azure-Sentinel/pull/3056/commits/3cd9e969e9cc37196062dc8b7aa528528bac4be5). Thanks!